edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/authenticators/DeviceScopeTokenAuthenticator.cs

```diff
@@ -61,13 +61,13 @@ protected override (Option<string> deviceId, Option<string> moduleId) GetActorId
             if (parts.Length == 3)
             {
                 hostName = parts[0];
-                deviceId = parts[2];
+                deviceId = WebUtility.UrlDecode(parts[2]);
```

Comment:
nit: should it be better to use regex to get device Id and module Id?

Reply:
hmm...probably, but this is how it worked before so I didn't bother changing it.